### PR TITLE
Order returned Event Types by name

### PR DESCRIPTION
**Pull request by a1exsh**
Better late than never.

### CODE REVIEW

File: core-metastore/src/main/java/org/zalando/nakadi/repository/db/EventTypeRepository.java [hunk 1]
```diff
@@ -109,7 +109,8 @@ public EventType mapRow(final ResultSet rs, final int rowNum) throws SQLExceptio
                         "FROM zn_data.event_type," +
                         "jsonb_to_recordset(et_event_type_object->'authorization'->'writers') " +
                         "AS writers(data_type text, value text) " +
-                        "WHERE writers.data_type = ? AND writers.value = ?",
+                        "WHERE writers.data_type = ? AND writers.value = ? " +
+                        "ORDER BY et_name",
```
Comment (adyach) [inline core-metastore/src/main/java/org/zalando/nakadi/repository/db/EventTypeRepository.java:113]: can we have it as an option in the api? `sort_by=X`, X is event type name, owning_application etc

Comment (a1exsh) [reply core-metastore/src/main/java/org/zalando/nakadi/repository/db/EventTypeRepository.java:113]: Not sure that brings a lot of value.  Filter by `owning_application` would be more appreciated, I guess.

**Comment by adyach** [comment]
👍

**Comment by antban** [comment]
:+1:

**Pull request merged**